<launch>
	<node name="pid_node" pkg="pid" type="controller" output="screen" >
      <param name="Kp" value="3.9" />
      <param name="Ki" value="5.9" />
      <param name="Kd" value="7.9" />
      <param name="rate" value="20" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="5" />
      <param name="cutoff_frequency" value="70" />
      <param name="topic_from_controller" value="c2p" />
      <param name="topic_from_plant" value="p2c" />
      <param name="node_name" value="left_wheel" />
    </node>
</launch>
